### PR TITLE
Add postman, newman and github pages deployment

### PR DESCRIPTION
**Pull request by OR13**
Adds postman tests and automated reporting.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -145,4 +145,4 @@ vscode
 
 workspace/
 *.cbor
-*.cose
+# *.cose
```
Comment (letmaik) [inline .gitignore:148]: Feel free to remove this line.

File: postman/run-sanity.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
```
Comment (letmaik) [inline postman/run-sanity.sh:1]: Shebang missing

File: postman/sanity.postman_collection.json [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Submit Claim",
+			"request": {
+				"method": "POST",
+				"header": [],
+				"body": {
+						"mode": "formdata",
```
Comment (letmaik) [inline postman/sanity.postman_collection.json]: The submit API isn't formdata, should be sent as-is in the body with header `application/cose`.

File: postman/sanity.postman_collection.json [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+								{
+										"key": "file",
+										"type": "file",
+										"enabled": true,
+										"src": "claim.cose"
+								}
+						]
+				},
+				"url": {
+					"raw": "http://127.0.0.1:8000/entries",
```
Comment (letmaik) [inline postman/sanity.postman_collection.json:26]: Looks like a lot of duplication here? Is there a short form for "url"?

Comment (OR13) [reply postman/sanity.postman_collection.json:26]: you can use variable names, that is probably a thing to do, after the basic tests are running in CI.

**Comment by OR13** [comment]
Proof that it works locally... 

Screen Shot 2023-03-07 at 1 45 12 PM

**Comment by OR13** [comment]
Progress, create is now working in CI...

Screen Shot 2023-03-07 at 1 46 02 PM

**Comment by OR13** [comment]
Retrieve claim working locally in postman....
Screen Shot 2023-03-07 at 1 47 04 PM

File: postman/sanity.postman_collection.json (hunk 4 above)
Comment (OR13) [reply postman/sanity.postman_collection.json:26]: deferring for now, these will become parameters later.

**Review by letmaik — APPROVED** [review]

**Pull request merged**